src/matrix/net/common.ts

```diff
@@ -40,8 +48,8 @@ export function encodeBody(body) {
         return {
             mimeType: "application/json",
             body: json,
-            length: body.length
-        };
+            length: json.length
```

Comment:
hmm, this is actually also incorrect ... IIRC this gives the character length, not the byte length. But given https://github.com/vector-im/hydrogen-web/pull/588/commits/dfd6a3be03b6afad66bba5a41a0906c79965b4a8, we can probably just remove the length property, it's not used anymore.